engine/docker-overview.md

```diff
@@ -267,8 +267,8 @@ the future, Docker may support other container formats by integrating with
 technologies such as BSD Jails or Solaris Zones.
 
 ## Next steps
-- Read about [Installing Docker Engine](installation/index.md#installation).
-- Get hands-on experience with the [Get Started With Docker](getstarted/index.md)
+- Read about [Install Docker Engine](installation/index.md#installation).
```

Comment:
This should be just "installing Docker" as the link text. We are preferring "Docker" over "Docker Engine" now. You don't need to mirror the actual topic's title exactly if you can fit it into the sentence.